src/vuex/modules/session.js

```diff
@@ -241,14 +238,7 @@ export default ({ apollo }) => {
       dispatch(`storeLocalPreferences`)
     },
     async getPreferredCurrency() {
```

Comment:
remove. isn't needed. just get the value right away

Reply:
Done